src/pytom_tm/entry_points.py

```diff
@@ -1008,6 +1019,18 @@ def match_template(argv=None):
             )
         )
         per_tilt_weighting = True
+
+    elif args.warp_xml_file is not None:
+        voxel_size, tilt_angles, dose_accumulation, ctf_params = (
+            parse_warp_xml_data(
+                args.warp_xml_file,
+                args.tomogram,
+                phase_flip_correction=phase_flip_correction,
+            )
+        )
+    defocus_handedness = args.defocus_handedness
+    per_tilt_weighting = True
```

Comment:
I forgot to add these two in the indentation comment as well :man_facepalming: 
```suggestion
        defocus_handedness = args.defocus_handedness
        per_tilt_weighting = True
```

Reply:
I will commit this one myself